src/domain/build.js

```diff
@@ -30,7 +36,7 @@ class Build {
   }
 
   get id() {
-    return `${this.flavor}-${this.image_type}-${this.registry}-${this.repository}`
+    return `${this.flavor}-${this.image_type}-${this.registry}-${this.repository}-${this.image_tag}`
```

Comment:
`get id`? We don't have an id for every summary element, we have all of this parameters to compare but... is this comparison an id :thinking: ?